### PR TITLE
added index file

### PR DESCRIPTION
**Pull request by gordon-qcode**

### CODE REVIEW

File: public_html/pixels2.css [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+}
+
+.navbar-menu {
+    /* Site navigation menu */
+    float: right;
+}
+
+.navbar-menu__item {
+    /* Each menu item */
+    line-height:67px;
```
Comment (achisholm) [inline public_html/pixels2.css]: A space between property and value here please

File: public_html/pixels2.css [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    vertical-align: top;
+    min-height: 415px;
+    padding-right: 35px;
+    position: relative;
+}
+
+.main-article__image {
+    /* main image in the main article */
+    float: left;    
+    vertical-align: top;
```
Comment (achisholm) [inline public_html/pixels2.css]: `vertical-align: top` is this necessary?

File: public_html/pixels2.css [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+    padding-right: 30px; /*padding here instead of content for better alignment at small screen widths */
```
Comment (achisholm) [inline public_html/pixels2.css]: Not sure this comment makes sense with the current implementation. Maybe just remove?

File: public_html/pixels2.css [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+}
+
+.main-article__content {
+    /* container for all article content EXCEPT the image */
+    vertical-align: top;
```
Comment (achisholm) [inline public_html/pixels2.css]: Are either of these rules necessary?

File: public_html/pixels2.css [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+
+.navbar-menu__item:hover {
+    /* Hover over menu item */
+    text-decoration: underline;
+}
+
+.main-article {
+    /* Container for the main article of the site */
+    overflow: auto;
+    min-height: 415px;
```
Comment (achisholm) [inline public_html/pixels2.css]: > min-height: 415px;

I think we could safely remove this now that the button is positioned statically?

What is the purpose of `padding-right: 35px`? Can that get removed now too?

Comment (gordon-qcode) [inline public_html/pixels2.css]: The purpose of both the min-height and the padding-right is to make small adjustments to the layout in order to closely mimic the prototype design image... but turning those rules off I notice now only makes a difference of only a few pixels so maybe it's better to get rid of them for code cleanness...... I'll make another commit.

Comment (achisholm) [inline public_html/pixels2.css]: For the `.main-article`, I think we should aim to have equal padding left and right, so could use the `.rounded-box` padding of 28px or if you wanted to faithfully replicate the visual you could override this with padding left/right on `.main-article` of 35px. Does that sound reasonable?

**Pull request merged**